python-project-template/.pre-commit-config.yaml.jinja

```diff
@@ -84,6 +84,25 @@ repos:
         # pre-commit's default_language_version, see
         # https://pre-commit.com/#top_level-default_language_version
         language_version: python3.10
+{% endif %}
+{% if mypy_type_checking != 'none' %}
+    # Analyze type hints and report errors.
+  - repo: local
+    hooks:
+      - id: mypy
+        name: mypy (python files in src/ and tests/)
+        entry: mypy
+        language: system
+        types: [python]
+        files: ^(src|tests)/
```

Comment:
Also not necessary to change here, but does it make sense to include `.../tests/` here, given that the GitHub workflow only covers `.../src/`? Or vice versa, should the workflow include `.../tests/`?

Reply:
That's a good point. It probably does make sense to include the tests in the workflow, I'll add that